Makefile

```diff
@@ -322,6 +330,6 @@ sync-deps:
 # "make mode" option.  $(O) is meant as a shortcut for $(SPHINXOPTS).
 %: Makefile
 	@echo -e "Specify one of the following supported build outputs"
-	@echo -e "- make linux\n- make macos\n- make windows\n- make k8s\n- make container"
+	@echo -e "- make linux\n- make macos\n- make windows\n- make k8s\n- make openshift\n- make container"
```

Comment:
Add `make all` as well?

```
all: linux macos windows k8s openshift container
```

Reply:
that is not done on purpose @kaovilai - we will be adding `make all` once some other things are cleaned up.